### DISCUSSION TITLE
不具合かも: ToCが突き抜ける

### DISCUSSION THREAD

**Discussion by Zect3279** [increments/qiita-discussions#118] [要望・アイデア・不具合]
TOC: 右サイドバー下部に表示されている目次

長い記事を開くと、TOCの下の方が切れます。
ページの一番下に行くとTOCの最後の部分まで見ることができます。
が、そこそこ不便ですし、多分不具合だと思うので、報告します。

**Comment by kyntk** [comment]
@sas08 ご報告ありがとうございます。
実は分かりにくいのですが、ToCはスクロール可能でして、スクロールすることで最後まで見ることができます。

ただ、一見スクロールできなそうなので気づかない方も多いのではないかと思うので、UIの改善を検討します!

**Reply by kyntk** [reply]
すみません！

追加で調査をしたところ、ウィンドウサイズが一定以下だと、ToCが突き抜けてしまっていることを確認できました。
こういうことでお間違い無いでしょうか？

![image](https://user-images.githubusercontent.com/17251829/151097887-6ea754fe-fda9-48e1-a849-dd4278a8ae41.png)


こちらは不具合になるので対応いたします！

ご報告ありがとうございます！

**Reply by Zect3279** [reply]
そうです！
tocが突き抜けてしまうことを言っていました。
わかりにくくてすみません、、
スクショを貼るべきでした

お願いします。

**Reply by kyntk** [reply]
@sas08 ありがとうございます！
issueを作って対応します！ https://github.com/increments/qiita-discussions/issues/120

**Reply by kskwtnk** [reply]
@sas08 
お待たせしました！
こちらの修正が完了しました :rocket: 

ご報告いただきありがとうございました！
またバグなどありましたら是非ご報告ください
